okio/src/commonMain/kotlin/okio/internal/-Utf8.kt

```diff
@@ -23,11 +23,12 @@ import okio.processUtf16Chars
 // to everything else. Putting them in this file, `-Utf8.kt`, makes them invisible to
 // Java but still visible to Kotlin.
 
-fun ByteArray.commonToUtf8String(): String {
-  val chars = CharArray(size)
+fun ByteArray.commonToUtf8String(offset: Int = 0, byteCount: Int = size): String {
```

Comment:
I would have expected (begin, end) instead of (offset, count) here, but I suspect you’re tracking something else

Reply:
I think this is probably a wash either way. Either I have to calculate the end index for (offset, count) or I have to calculate the size for (begin, end). I'll change it for the follow up PR though, just in case you like it better.